tests/e2e/backup_restore_suite_test.go

```diff
@@ -295,8 +295,16 @@ var _ = ginkgo.Describe("Backup and restore tests", ginkgo.Ordered, func() {
 		// using kopia to collect more info (DaemonSet)
 		waitOADPReadiness(lib.KOPIA)
 
+		log.Printf("Creating real DataProtectionTest before must-gather")
+		bsls, err := dpaCR.ListBSLs()
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+		bslName := bsls.Items[0].Name
+		err = lib.CreateUploadTestOnlyDPT(dpaCR.Client, dpaCR.Namespace, bslName)
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
```

Comment:
need to also add this to virt e2er tests https://github.com/openshift/oadp-operator/blob/88b603e60b54181fde0cda2e3ea04cd4f5df5535/tests/e2e/virt_backup_restore_suite_test.go#L220